django_rq/workers.py

```diff
@@ -23,3 +23,20 @@ def get_worker(*queue_names):
     return Worker(queues,
                   connection=queues[0].connection,
                   exception_handlers=get_exception_handlers() or None)
+
+
+def collect_workers_per_configuration(queue_list):
```

Comment:
Just remembering that the changes were implemented and are waiting for approval. :)